locale/en.po

```diff
@@ -1674,6 +1674,16 @@ msgstr ""
 "version.\n"
 "Please load the save manually through the menu."
 
+#: ../src/gui_common/charts/line/LineChart.cs:900
+#: ../src/gui_common/charts/line/LineChart.cs:929
+msgid "MAX_VISIBLE_DATASET_WARNING"
+msgstr "Not allowed to show more than {0} datasets!"
+
+#: ../src/gui_common/charts/line/LineChart.cs:909
+#: ../src/gui_common/charts/line/LineChart.cs:937
+msgid "MIN_VISIBLE_DATASET_WARNING"
+msgstr "Not allowed to hide less than {0} dataset(s)!"
```

Comment:
Should this be actually hide or show here?

Reply:
Ah, had a brainfart there. I think this should be `Not allowed to hide more than..`